dlc-manager/src/sub_channel_manager.rs

```diff
@@ -895,6 +1052,82 @@ where
         Ok(())
     }
 
+    /// Notify that LDK has decided to close the channel with given id.
```

Comment:
I think we should document that this MUST be called in the LDK event handler.

Reply:
Done